### DISCUSSION TITLE
error in 1st line of code

### DISCUSSION THREAD

**Discussion by MAtabak1** [AammarTufail/python_ka_chilla2023#114] [General]
C:\WINDOWS\system32>cd Desktop
The system cannot find the path specified.

C:\WINDOWS\system32>
what is this error

**Comment by RizwanAle** [comment]
what you're trying to do is not accurate, try below one to change directory. Copy the below line and past in into cmd. 
cd /d %userprofile%/desktop use this one to switch on Desktop.
